dist/blocks/scaffolding/examples/notifications-phase-banner.html

```diff
@@ -42,9 +42,12 @@
   </script>
   <div class="nhsuk-hee-phasebanner">
     <div class="nhsuk-width-container">
-      <p class="nhsuk-hee-phasebanner__content">
-        <span class="nhsuk-hee-phasebanner__tag"> Alpha </span> This is a new service – your <a class="nhsuk-link" href="https://forms.office.com/Pages/ResponsePage.aspx?id=K5Gn_5ewMUGcD9DoB1Wyq62_imvDlFpPhawBoObID1NUQTNENlpXMlNZVjJBS0JCUVVLUlQxSkJVUyQlQCN0PWcu" target="_blank" title="Opens in New Window">feedback</a> will help us to improve it.
-      </p>
+      <div class="nhsuk-hee-phasebanner__wrapper">
+        <span class="nhsuk-hee-phasebanner__tag"> Alpha </span>
+        <div class="nhsuk-hee-phasebanner__content">
+          <p>This is a new service – your <a class="nhsuk-link" href="https://forms.office.com/Pages/ResponsePage.aspx?id=K5Gn_5ewMUGcD9DoB1Wyq62_imvDlFpPhawBoObID1NUQTNENlpXMlNZVjJBS0JCUVVLUlQxSkJVUyQlQCN0PWcu" target="_blank" title="Opens in New Window">feedback</a> will help us to improve it.</p>
```

Comment:
Is it possible to apply `nhsuk-link` anchor link decoration via `nhsuk-hee-phasebanner__content` so that BE doesn't have to do it explicitly ?